server/cpho/views/indicator_data.py

```diff
@@ -96,6 +97,64 @@ class Meta:
         required=False, widget=forms.TextInput(attrs={"class": "form-control"})
     )
 
+    def clean(self):
+        cleaned_data = super().clean()
+        data = cleaned_data['value']
+        value_unit = cleaned_data['value_unit']
+        print(data)
+        if value_unit == "%":
+            if data and data > 100 or data < 0:
+                self.add_error("value", tdt("Value must be a percentage between 0-100"))
+        if data and data < 0:
```

Comment:
this "if" part should go inside clean_value() because this is cleaning that is specific to value independent of other fields